DayCare/app/models/child.rb

```diff
@@ -0,0 +1,14 @@
+class Child < ActiveRecord::Base
+    belongs_to :class_room
+    has_many :class_rooms, through: :waiting_list_entries
+
+		def add_to_wait_list(class_room)
+		end
+
+		def join_class_room(class_room)
+		end
+
+		def leave_class_room
```

Comment:
Thanks, but I don't understand why this is the one that should be deleted and not the analog/duplicate method I put on class_room.  Since the `children` table is the one with the foreign key, shouldn't the child model be the one with this method, since it will simply entail setting `children.class_room_id` to `nil`?